examples/chainer_simple.py

```diff
@@ -94,10 +94,13 @@ def objective(trial):
         ]))
     trainer.extend(log_report_extension)
 
+    trainer.extend(
+        ChainerPruningExtension(trial, 'validation/main/accuracy', (1, 'epoch')))
+
     # Run!
-    trainer.run()
+    trainer.run(show_loop_exception_msg=False)
 
-    # Set the user attributes such as loss and accuracy for train and validation sets
+# Set the user attributes such as loss and accuracy for train and validation sets
```

Comment:
Could you fix this indentation?
```suggestion
    # Set the user attributes such as loss and accuracy for train and validation sets
```